cruise_company_spring/src/main/java/com/example/cruise_company/service/repository/impl/LinerRepositoryListImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.example.cruise_company.service.repository.impl;
+
+import com.example.cruise_company.service.model.Liner;
+import com.example.cruise_company.service.repository.LinerRepository;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers